HorizontalPodAutoscaler/README.md

```diff
@@ -0,0 +1,51 @@
+To create the Deployment, apply the nginx.yaml manifest:
+
+> `kubectl apply -f nginx.yaml`
+
+To create an HPA object that only targets average CPU utilization, use the following command:
+
+> `kubectl autoscale deployment horizontal-pod-autoscaler-cpu-based --cpu-percent=50 --min=1 --max=10`
```

Comment:
why not 'kubectl apply -f hpa-multi.yaml'?

Reply:
To demonstrate another way to create a HPA object - manually by `kubectl autoscale` command.
Creating HPA object by manifest ('kubectl apply -f hpa-multi.yaml') used a bit later, after removing previously created HPA object:
`kubectl delete hpa horizontal-pod-autoscaler-cpu-based`
If `kubectl autoscale` unnecessary - I'll remove all related commands.